examples/conversationbot.py

```diff
@@ -46,7 +46,7 @@ def start(bot, update):
 
 def gender(bot, update):
     user = update.message.from_user
-    logger.info("Gender of %s: %s" % (user.first_name, update.message.text))
+    logger.info("Gender of {}: {}".format(user.first_name, update.message.text))
```

Comment:
Good catch here, we shouldn't be formatting logs ourselves but rather let logging do it. Unfortunately logging only work with `%s` notation. So please change to:
```
logger.info("Gender of %s: %s", user.first_name, update.message.text)
```
edit: I see that part of the logging occurrences you've done that way.